documentation/user_docs/docs/manual/Manipulating_and_extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -2,166 +2,367 @@
+   pixels from ``-1`` to ``1`` inclusive.
+
+* ``[lower,step,upper]``
+
+  A three-component binning axis specifies an axis is a plot axis with the first ``lower`` and the last ``upper``
+  components specifying the centres of the first and the last bins of the data to be cut. The middle component specifies
+  the bin width.
+
+.. note ::
+
+   If ``step`` is ``0``, the ``step`` is taken from the (current|previous) binning axes.
+
+.. warning::
+
+   A three-component binning axis defines the integration region by bin centres, i.e. the limits of the
+   data to be cut lie between ``min = lower-step/2`` and ``max = upper+step/2``, including ``min/max``
+   values. For example, ``[-1 1 1]`` will capture pixels from ``-1.5`` to ``1.5`` inclusive.
+
+
+* ``[lower, separation, upper, cut_width]``
+
+  A four-component binning axis defines multiple cuts with multiple integration limits in the selected direction.
+  These components are:
+
+  * ``lower``
+
+    minimum cut centre
+
+  * ``separation``
 
-This is a specific case of cut_sqw that can be used to take cuts from sqw objects that are already in memory, rather than on disk in the SQW file. Such objects can either be the result of using cut_sqw, or if have been read into memory from disk using the read_horace function
+    distance between cut centres
+
+  * ``upper``
+
+    maximum cut centre
+
+  * ``cut_width``
+
+    half-width of each cut from each bin-centre in both directions
+
+  The number of cuts produced will be the number of ``separation``-sized steps between ``lower`` and ``upper``.
+
+
+.. warning::
+
+   ``upper`` will be automatically increased such that ``separation`` evenly divides ``upper - lower``.  For example, ``[106,
+   4, 113, 2]`` defines the integration ranges for three cuts, the first cut integrates the axis over ``105-107``, the
+   second over ``109-111`` and the third ``113-115``.
+
+
+Optional arguments
+------------------
+
+.. code-block:: matlab
+
+   my_cut = cut (data_source, proj, p1_bin, p2_bin, p3_bin, p4_bin, '-nopix', filename)
+
+
+* ``'-nopix'``
+
+  means that the individual pixel information contributing to the resulting data is NOT retained (at present the default
+  is to retain it, resulting in an output that is an sqw object, whereas using ``'-nopix'`` gives a dnd output).
+* ``filename``
+
+  is a string specifying a full filename (including path) for the data to be stored, in addition to being stored in the
+  Matlab workspace.
+
+Further Examples
+----------------
 
 To take a cut from an existing sqw or dnd object, retaining the existing projection axes and binning:
 
-::
+.. code-block:: matlab
+
+   w1 = cut(w,[],[lo1,hi1],[lo2,hi2],...)
+
+.. note::
+
+   The number of binning arguments need only match the dimensionality of the object ``w`` (i.e. the number of plot
+   axes), so can be fewer than 4.
+
+.. note::
+
+   You cannot change the binning in a dnd object, i.e. you can only set the integration ranges and have to use ``[]``
```

Comment:
```suggestion
   You cannot change the binning in a dnd object, i.e. you can only set the integration ranges, and have to use ``[]``
```

Presuming this is for cutting a dnd object. Here you don't have a set of pixels to rebin, and can only combine existing dnd bins. Do integration ranges still use the existing bins and does changing the range imply contracting the number of bins used?